lib/r2d2/util.rb

```diff
@@ -44,8 +45,10 @@ def verify_mac(mac_key, encrypted_message, tag)
       raise TagVerificationError unless secure_compare(mac, Base64.decode64(tag))
     end
 
-    def decrypt_message(encrypted_data, symmetric_key)
-      decipher = OpenSSL::Cipher::AES128.new(:CTR)
+    def decrypt_message(encrypted_data, symmetric_key, cipher_key_length_bits = 128)
+      raise ArgumentError, "Invalid cipher_key_length #{cipher_key_length_bits} must be 128 or 256" unless [128, 256].include?(cipher_key_length_bits)
+
+      decipher = cipher_key_length_bits == 256 ? OpenSSL::Cipher::AES256.new(:CTR) : OpenSSL::Cipher::AES128.new(:CTR)
```

Comment:
The following reads easier and also doesn't use deprecated constants:
```suggestion
      decipher = OpenSSL::Cipher.new("aes-#{cipher_key_length_bits}-ctr")
```